Gemfile

```diff
@@ -38,3 +38,7 @@ else
   end
 end
 # END ENGINE_CART BLOCK
+
+unless File.exist?(file)
+  eval_gemfile File.expand_path('spec/test_app_templates/Gemfile.extra', File.dirname(__FILE__))
```

Comment:
Can we inject the gem via the test app generator like this? https://github.com/projectblacklight/blacklight/blob/610310f89e0763d7632f861328c3828139415bc3/lib/generators/blacklight/assets_generator.rb#L29-L31